spec/features/withdrawing/withdraw_spec.rb

```diff
@@ -32,10 +32,12 @@ def and_i_fill_in_the_explanation
   end
 
   def and_i_confirm_the_withdrawal
-    converted_explanation = GovspeakDocument.new(@explanation, @edition).payload_html
-    body = { type: "withdrawal", explanation: converted_explanation, locale: @edition.locale }
-    stub_publishing_api_unpublish(@edition.content_id, body: body)
-    click_on "Withdraw document"
+    freeze_time do
+      converted_explanation = GovspeakDocument.new(@explanation, @edition).payload_html
+      body = { type: "withdrawal", explanation: converted_explanation, locale: @edition.locale, unpublished_at: Time.zone.now }
```

Comment:
It might be time to split this hash over a number of lines to make it a bit more readable.